apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -41,6 +45,7 @@ export function LoginForm({}: Props) {
       const response = await mutateAsync(itemData);
 
       setToken((response as any).token);
+      if (isFromVercel) return;
       navigate('/templates');
```

Comment:
I would simplify these two lines into `if (!isFromVercel) navigate('/templates');`

Reply:
I would add `{}` and then I think this is fine because it is easier to read even if it makes more code.